src/core/ipv4/autoip.c

```diff
@@ -224,8 +224,9 @@ autoip_conflict_callback(struct netif *netif, acd_callback_enum_t state)
       break;
     case ACD_DECLINE:
       /* "delete" conflicting address so a new one will be selected in
```

Comment:
This comment says _start will select a new address. Can we make it do that instead? If we increment it every time in `autoip_start` it should also pass the tests right?

Reply:
Or alternatively, edit the comment to match the new way of handling it.